custom_functions/regex_extract_ipv4.py

```diff
@@ -0,0 +1,31 @@
+def regex_extract_ipv4(input_string=None, **kwargs):
+    """
+    Takes a single input and attempts to extract all IPv4 addresses from it using regex.
+    
+    Args:
+        input_string (CEF type: *): An input string that may contain an arbitrary number of ipv4 addresses
+    
+    Returns a JSON-serializable object that implements the configured data paths:
+        *.ipv4 (CEF type: ip): Extracted ipv4 addresses
+    """
+    ############################ Custom Code Goes Below This Line #################################
+    import json
+    import phantom.rules as phantom
+    import re
+    
+    outputs = []
+    ip_list = []
+    for ip in input_string:
+        if ip:
+            ip_rex = re.findall('(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)',str(ip))
```

Comment:
At the end of this line I think the `str(ip)` is causing issues and should be replaced with just `ip`. The way it is currently does not handle non-ascii unicode characters and throws an exception like:

```UnicodeEncodeError: 'ascii' codec can't encode character u'\xfc' in position 218: ordinal not in range(128)```